src/zope/paste/tests.py

```diff
@@ -124,14 +124,27 @@ def test_basic_serve():
 def test_serving_test_app():
     """Test serving a real app.
 
+    Setup logging, so we get the message ``Serving on ...`` into our doctest
+    output:
+
+    >>> import logging
+    >>> logger = logging.getLogger('waitress')
+    >>> logger.addHandler(logging.StreamHandler(sys.stdout))
+    >>> logger.setLevel(logging.INFO)
+
+    Start the app:
+
+    >>> import os
+    >>> import os.path
     >>> import threading
     >>> from zope.paste import serve
-    >>> server = threading.Thread(target=serve.serve, args=(['app.ini'],))
+    >>> app_ini = os.path.join(os.getcwd(), 'app.ini')
```

Comment:
Isn't this better expressed as 
```suggestion
    >>> app_ini = os.path.abspath('app.ini')
```
?

Reply:
Let's try it.